.github/workflows/Unit-Test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
             cd ~ && \
             git clone --depth=1 https://github.com/chromebrew/chromebrew.git build_test && \
             cd build_test && \
-            yes | CREW_CACHE_ENABLED=1 crew build -f packages/hello_world_chromebrew.rb"
+            yes | CREW_CACHE_ENABLED=1 crew build -vf packages/hello_world_chromebrew.rb"
```

Comment:
Make verbose optional?

Reply:
It is just there to make sure verbose mode works.

(Verbose mode is disabled for the armv7l run of the unit test.)

Reply:
Yeah, that makes sense.  I just realized this was for tests.